packages/office-ui-fabric-react/src/components/Autofill/Autofill.tsx

```diff
@@ -160,21 +160,24 @@ export class Autofill extends BaseComponent<IAutofillProps, IAutofillState> impl
   // Find out more at https://developer.mozilla.org/en-US/docs/Web/Events/compositionstart
   private _onCompositionUpdate = () => {
     if (isIE11()) {
-      this._updateValue(this._getCurrentInputValue());
+      this._updateValue(this._getCurrentInputValue(), true);
     }
   };
 
   // Composition events are used when the character/text requires several keystrokes to be completed.
   // Some examples of this are mobile text input and langauges like Japanese or Arabic.
   // Find out more at https://developer.mozilla.org/en-US/docs/Web/Events/compositionstart
   private _onCompositionEnd = (ev: React.CompositionEvent<HTMLInputElement>) => {
+    this._isComposing = false;
     const inputValue = this._getCurrentInputValue();
     this._tryEnableAutofill(inputValue, this.value, false, true);
     this._isComposing = false;
     // Due to timing, this needs to be async, otherwise no text will be selected.
     this._async.setTimeout(() => {
-      // Call getCurrentInputValue here again since there can be a race condition where this value has changed during the async cal
-      this._updateValue(this._getCurrentInputValue());
+      // it's technically possible that the value of _isComposing is reset during this timeout,
+      // so explicitly trigger this with composing=true here, since it is supposed to be the
+      // update for composition end
+      this._updateValue(this._getCurrentInputValue(), true);
```

Comment:
Since it's the end, isn't composing false at this point? Composition has completed and this is final value that should be used?

Reply:
Anyway, I think this is fine, but I'm going to move from `this._isComposing` to `(ev.nativeEvent as any).isComposing` where possible, since I think in the above edge case _isComposing can't be trusted.